pkg/machines/libvirt-dbus.js

```diff
@@ -196,6 +196,19 @@ LIBVIRT_DBUS_PROVIDER = {
         return () => call(connectionName, vmId, 'org.libvirt.Domain', 'AttachDevice', [xmlDesc, flags], TIMEOUT);
     },
 
+    CHANGE_NETWORK_AUTOSTART({
+        connectionName,
+        networkName,
+        autostart,
+    }) {
+        return (dispatch) => call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'NetworkLookupByName', [networkName], TIMEOUT)
```

Comment:
Why you need dispatch here?

Actually this method is not using any redux reducers. So it doesn't even have to be in the virt wrapper. It can be in the bottom of the file with the other methods that are libvirt-dbus specific. And you can get rid of all the wrappers.

Reply:
Got rid of dispatch and wrappers.